query-algebrizer/src/types.rs

```diff
@@ -294,6 +295,7 @@ pub enum ColumnConstraint {
         right: QueryValue,
     },
     HasType(TableAlias, ValueType),
+    NotExists(ComputedTable),
```

Comment:
This also confused me — I checked out your branch and this part of the diff isn't there!